.github/workflows/BuildPR.yml

```diff
@@ -43,7 +43,11 @@ jobs:
         uses: actions/upload-artifact@v3
         if: ${{ success() }}
         with:
-          name: GitX built by ${{ matrix.xcode }}-${{ matrix.abi }}
-          path: |
-            GitX-${{ matrix.abi }}.dmg
-            GitX-${{ matrix.abi }}.zip
+          name: GitX-${{ matrix.abi }}.dmg
+          path: GitX-${{ matrix.abi }}.dmg
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        if: ${{ success() }}
```

Comment:
Hmm, I'm not sure, if this `if: ${{ success() }}` is related to previous step, and not to the one more previous prepare step. Anyway, we can give it a try